.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
```

Comment:
`TypedArrays` have been an integral part of Cesium since the very beginning and are an important part of what makes WebGL possible, so your notion that they were added in `1.29` is incorrect.  Even `Float64Array` specifically has been used in Cesium for years.

My confusion here stems from the fact that they existing well-before ES6 but were not part of the formal ECMA specification until ES6.  So browsers support has been around forever based on an older spec (see the chart at the bottom of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays).

It turns out that `jsHint` has a separate setting for dealing with typed arrays outside of the version of ECMAScript being used, and from what I can tell eslint does not (I found a few GitHub issues of people complaining about this because it was a breaking change at some point on their end).

I am tempted to say just use the es6 toggle, but let's start with your globals suggestion and see where it goes.  That's the safer choice for now and we can revisit when we migrate to ES6 modules.